live-examples/css-examples/backgrounds-and-borders/background.html

```diff
@@ -0,0 +1,44 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="background">
+<div class="example-choice">
+<pre><code class="language-css">background: green;</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+<div class="example-choice">
+<pre><code class="language-css">background: content-box radial-gradient(crimson, skyblue, khaki);</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+<div class="example-choice">
+<pre><code class="language-css">background: no-repeat url("../../media/examples/lizard.png");</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+<div class="example-choice">
+<pre><code class="language-css">background: left 5% / 15% 60% repeat-x url("../../media/examples/star.png");</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+<div class="example-choice" initial-choice="true">
+<pre><code class="language-css">background: center / contain no-repeat url("../../media/examples/firefox-logo.svg"),
+            #cf4 35% url("../../media/examples/lizard.png");</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
```

Comment:
This one is a bit...lively. I wonder if we should tone down the colours a bit. But that's just a subjective opinion.